### PR TITLE
How to work with programmers on e-commerce projects

### PR DESCRIPTION
**Pull request by sunmeng0421**
Discussing the process of managing developer teams when launching a new e-commerce website.

### CODE REVIEW

**Comment by tingtingguo** [comment]
halp...I don't quite know what I'm doing

**Comment by sunmeng0421** [comment]
@tingtingguo , you need to create a pull request so that we can comment on your branch.

File: how-to-launch-ecommerce-website .md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Before CBS, I was a digital marketing manager at a Fortune 500 transportation company, my role is to lead a e-commerce website launch. As a business manager, I worked closely with my counterpart - Director of IT platform to launch a website which sells over 10,000 trucks online. The project was successfully luanched within deadline and I found there are particularly important learnings we can take from th Digital Literacy Class.
```
Comment (tingtingguo) [inline how-to-launch-ecommerce-website .md]: @sunmeng0421 small typo on this line - change "luanched" to "launched"

Comment (tingtingguo) [inline how-to-launch-ecommerce-website .md]: Overall looks really good @sunmeng0421 I especially liked the link that you included at the end, and how you incorporated your own experience into the summary! :+1:

**Comment by sunmeng0421** [comment]
thanks, @tingtingguo fixed!

File: how-to-launch-ecommerce-website .md (hunk 1 above)
Comment (sunmeng0421) [inline how-to-launch-ecommerce-website .md]: thanks, @tingtingguo fixed!

**Comment by SeanOMealia** [comment]
Hi @sunmeng0421 -- I love it!  Well done.  At one point you say "you can't always get what you want".  I think the essay would be a little stronger if you hyperlinked to the song below when you say that.

https://www.youtube.com/watch?v=7S94ohyErSw

**Comment by tingtingguo** [comment]
@SeanOMealia  are you in Watson?  I can show you

**Comment by tingtingguo** [comment]
@sunmeng0421 @SeanOMealia  are you good to ship?

**Comment by sunmeng0421** [comment]
yup, ready to ship!

**Pull request merged**